### DISCUSSION TITLE
Can't deactivate AdGuard Home Filter temporarily

### DISCUSSION THREAD

**Discussion by PKM85** [hassio-addons/addon-adguard-home#299] [Q&A]
Hi everyone! 
I have a problem with the HA AdGuard Home Addon: Unfortunately I cannot deactivate the filter once the addon is running. Sometimes you obviously need to disable the blocker for a period of time - but I can't. The only way is to change the DNS server on the router back to standard. Deactivating the Addon in Homeassistant (via the Integration "Switch", or in the Addon UI) changes nothing... does anyone have an idea?
Thanks in advance! 
Patrick

**Comment by m0ngr31** [comment]
I have to use the `AdGuard Protection` switch
